solutions/instances/outputs.tf

```diff
@@ -14,22 +14,22 @@ output "resource_group_id" {
 
 output "scc_id" {
   description = "SCC instance ID"
-  value       = module.scc.id
+  value       = var.existing_scc_instance_crn == null ? module.scc[0].id : null
```

Comment:
The ID is the CRN - so you can get this from `var.existing_scc_instance_crn` instead of defaulting to `null`

Reply:
addressed